src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevService.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.familie.ks.sak.kjerne.brev
+
+import no.nav.familie.kontrakter.felles.arbeidsfordeling.Enhet
+import no.nav.familie.ks.sak.api.dto.ManueltBrevDto
+import no.nav.familie.ks.sak.api.dto.tilBrev
+import no.nav.familie.ks.sak.common.exception.Feil
+import no.nav.familie.ks.sak.kjerne.arbeidsfordeling.ArbeidsfordelingService
+import no.nav.familie.ks.sak.kjerne.brev.domene.maler.Brev
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.PersonopplysningGrunnlagService
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+
+@Service
+class BrevService(
+    private val brevKlient: BrevKlient,
+    private val personopplysningGrunnlagService: PersonopplysningGrunnlagService,
+    private val arbeidsfordelingService: ArbeidsfordelingService
+) {
+
+    fun hentForhåndsvisningAvBrev(behandlingId: Long, manueltBrevDto: ManueltBrevDto): ByteArray {
+        val mottakerPerson = personopplysningGrunnlagService.hentSøker(behandlingId)
+        val arbeidsfordelingPåBehandling = arbeidsfordelingService.hentArbeidsfordelingPåBehandling(behandlingId)
+
+        val manueltBrevDtoMedMottakerData = manueltBrevDto.copy(
+            enhet = Enhet(
+                enhetNavn = arbeidsfordelingPåBehandling.behandlendeEnhetNavn,
+                enhetId = arbeidsfordelingPåBehandling.behandlendeEnhetId
+            ),
+            mottakerMålform = mottakerPerson?.målform ?: manueltBrevDto.mottakerMålform,
+            mottakerNavn = mottakerPerson?.navn ?: manueltBrevDto.mottakerNavn
+        )
+        return genererManueltBrev(manueltBrevDtoMedMottakerData, true)
+    }
+
+    private fun genererManueltBrev(
+        manueltBrevRequest: ManueltBrevDto,
+        erForhåndsvisning: Boolean = false
+    ): ByteArray {
+        Result.runCatching {
+            val brev: Brev = manueltBrevRequest.tilBrev()
```

Comment:
```suggestion
            val brev = manueltBrevRequest.tilBrev()
```
Kommer fram av navnet på extension function hva slags type det er 👍